client/components/RatingsAndReviews/Reviews/Review.jsx

```diff
@@ -1,12 +1,65 @@
 import React from 'react';
 import Rating from '@material-ui/lab/Rating';
+import { makeStyles, Typography, Grid, Divider } from '@material-ui/core';
+// import Grid from '@material-ui/core/Grid';
+// import Typography from '@material-ui/core/Typography';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexGrow: 1,
+  },
+  reviewTitle: {
+    fontWeight: 500,
+    fontSize: 16
+  },
+  userDateHelp: {
+    fontSize: 12,
+    color: "grey"
+  },
+  reviewBody: {
+    color: "#777272",
+    fontSize: 14
+  },
+  section: {
+    margin: theme.spacing(2)
+  },
+  dividerLine: {
+    backgroundColor: "black"
+  }
+
+}));
 
 const Review = () => {
+  const classes = useStyles();
   return (
-    <>
-    <Rating name="read-only" readOnly />
-    <p>Review title with word-break trancation to prevent</p>
-</>
+    <div className={classes.root}>
+      <Grid container spacing={3}>
+        <Grid item xs={3}>
+        <Rating name="read-only" readOnly />
+        </Grid>
+        <Grid item xs={3}>
```

Comment:
Everything looks great!  Nit: looks like you could combine these lines into one section with xs={6} rather than two of the same thing with xs={3}:
      <Grid item xs={6}>
          {/* This is just to keep spacing between two items in this row */}
        </Grid>

Super minor though, so will approve!

Reply:
Ah yesss, thank you for pointing this out. It's a quick one. I'll update